website/docs/Installation/macos-install.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 3
+---
+
+# MacOS
+## Installing Podman Desktop on MacOS using .dmg file
+
+If you already have a Podman machine up and running, you can check out the [Downloads](/downloads/macos) section of this website to get the applicable .dmg file depending on your Mac Hardware Architecture (i.e. Intel or Apple M1).
```

Comment:
you only need podman being installed ( at least v4) but no need for a running machine. You can create it from Desktop

Also there is the Universal binary that is easier as you don't need to know your arch

Reply:
so i should just delete the step 2?